backend/src/attraction/services/attraction_service.py

```diff
@@ -1,49 +1,82 @@
+import json
 import logging
 
+from fastapi import HTTPException, Response
 from sqlalchemy.orm import Session
 
 from src.attraction.interfaces.repository import Repository
-from src.attraction.schemas import AttractionSchema, AttractionImages
-from src.core.cloudstorage import CloudStorage
+from src.attraction.schemas import AttractionImages, AttractionSchema
+from src.core.json_encoder import JSONEncoder
+from src.db.cache_storage import CacheStorage
+from src.db.cloudstorage import CloudStorage
 
 logger = logging.getLogger(__name__)
+cache = CacheStorage()
+cloud_storage = CloudStorage()
```

Comment:
Just saying, if you would like to test it in the future this would make the task much harder

Reply:
If you want to test this service with units you can mock response from it, I dont see any difficulties